src/constraints/scalaraffine.jl

```diff
@@ -154,7 +154,7 @@ function MOI.get(m::LinQuadOptimizer, ::MOI.ConstraintFunction, c::LCI{<: LinSet
     # TODO (@joaquim) fix indexing
     terms = map(
         (v,c)->MOI.ScalarAffineTerm{Float64}(c,v),
-        m.variable_references[colidx+1],
+        m.variable_references[colidx],
```

Comment:
@odow 1 indexing here, need to fix GLPK and Gurobi

Reply:
Cool. I knew there were probably things like this. It gets too confusing switching between all the different formats and 0- and 1-indexing.